### PR TITLE
Feature/boyar compat

### PR DESCRIPTION
**Pull request by uv-orbs**

### CODE REVIEW

File: deployment/nginx/conf.d/default.conf [hunk 1]
```diff
@@ -0,0 +1,40 @@
+server {
+    listen 80;
+    server_name _;
+    # rewrite all plural service(s) and log(s) to a single form
+    location ~* ^/services/(.*)$ {
+        rewrite ^/services/(.*)$ /service/$1 last;         
+    }
+    # location ~* ^/services/(.*)/logs {
```
Comment (Luke-Rogerson) [inline deployment/nginx/conf.d/default.conf:8]: Should these comments be removed?

File: manager/manager.py [hunk 2]
```diff
@@ -1,15 +1,53 @@
...
+    "currentVersion": "0.0.0",
+    "scheduledVersion": None,
+    "updateScheduled": None,
+    "updateScheduledFor": None,
+}
+
+
+# logger
+def append_line_to(file_name, line):
+    print(line)
```
Comment (Luke-Rogerson) [inline manager/manager.py:30]: Do you mean to print every line?

File: manager/manager.py [hunk 3]
```diff
@@ -1,15 +1,53 @@
...
+data = {
+    "currentVersion": "0.0.0",
+    "scheduledVersion": None,
+    "updateScheduled": None,
+    "updateScheduledFor": None,
+}
+
+
+# logger
+def append_line_to(file_name, line):
```
Comment (Luke-Rogerson) [inline manager/manager.py:29]: Possible to add type args (see what I did for `run_command` for an example)?

File: manager/status.py [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+        # self.error = ""
+        return {
+            "Timestamp": self.timestamp,
+            "Status": self.status,
+            "Error": self.error,
+            "Payload": self.payload,
+        }
+
+
+# Usage
```
Comment (Luke-Rogerson) [inline manager/status.py:133]: Remove these?

File: deployment/nginx/conf.d/default.conf (hunk 1 above)
Comment (uv-orbs) [reply deployment/nginx/conf.d/default.conf:8]: most probably, i'll do it

File: manager/manager.py (hunk 2 above)
Comment (uv-orbs) [reply manager/manager.py:30]: std out doesnt go anywhere - its just nice if one debugs this file in their console

File: manager/manager.py (hunk 3 above)
Comment (uv-orbs) [reply manager/manager.py:29]: sure thing

File: manager/status.py (hunk 4 above)
Comment (uv-orbs) [reply manager/status.py:133]: i like to keep it for sake of debug, but not crucial

**Pull request merged**